### PR TITLE
Update external FHIR server config doc

### PR DESCRIPTION
**Pull request by srcarrier**
• Modify property to match singular LFH change
• Include https w/ user id/password example

### CODE REVIEW

**Comment by srcarrier** [comment]
This doc update corresponds to LinuxForHealth/connect#338

File: source/feature-configuration.rst [hunk 1]
```diff
@@ -33,24 +33,24 @@ Environment Variable::
...
 
 +-------------------------+----------------------------------------+---------------------+
 |                         | Name                                   | Default Value       |
 +=========================+========================================+=====================+
-| Property                | lfh.connect.fhir-r4.externalservers    | None                |
+| Property                | lfh.connect.fhir-r4.externalserver    | None                 |
 +-------------------------+----------------------------------------+---------------------+
-| Environment Variable    | LFH_CONNECT_FHIR-R4_EXTERNALSERVERS    | None                |
+| Environment Variable    | LFH_CONNECT_FHIR-R4_EXTERNALSERVER    | None                 |
 +-------------------------+----------------------------------------+---------------------+
```
Comment (ccorley) [inline source/feature-configuration.rst:46]: It looks like this table is off by a character or so - if I view the file in Github, I can see that the last column isn't being recognized.

**Review by ccorley — CHANGES_REQUESTED** [review]
Nice - this included a couple other required updates, so thanks for the refresh!  Just a minor table spacing change needed.

File: source/feature-configuration.rst (hunk 1 above)
Comment (srcarrier) [reply source/feature-configuration.rst:46]: Good catch 👍 - fixed

**Review by ccorley — APPROVED** [review]

**Pull request merged**